src/impi/core.cljs

```diff
@@ -251,6 +254,12 @@
      :obj (create-render-texture value)}
     {:val value, :obj (get-texture value)}))
 
+(defmethod update-prop! :pixi.tiling-sprite/tile-scale [sprite _ _ [x y]]
+  (set! (.-tileScale sprite) #js {:x x :y y}))
```

Comment:
Rather than creating a new object, instead you can update the scale and position directly. See the multimethods for `:pixi.object/position` and `:pixi.object/scale`. Also ensure that the scaling has a default of 1 so that it goes back to 1:1 scaling if the scale property is removed.